announcements/2020-Week-48/announcement.md

```diff
@@ -0,0 +1,24 @@
+# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
+2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.
+3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.
+
+## Web UI
+
+
+[Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplified deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) added support for upgrading to beta versions.
+
+## Documentation
+
+- We extended our guide on [automatic termination of unhealthy nodes](https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/) for Azure, where this function is available as tenant cluster release v13.1.0.
+- The article on the [operational layers](https://docs.giantswarm.io/basics/giant-swarm-operational-layers/) got an overhaul.
+- We added explanation for the use of deprecated tenant cluster releases where it matters.
+- The guide on [adding labels to tenant clusters](https://docs.giantswarm.io/guides/tenant-cluster-labelling/) has been updated to inform you that this is possible on Azure.
```

Comment:
@axbarsan was this released?

Reply:
I took these from docs release notes (https://docs.giantswarm.io/changes/documentation/), so the answer is yes

Reply:
> or v{{% first_azure_nodepools_version %}} and above for Azure

First azure nodepools version is v13.0.0 and it was not released yet

Reply:
Which docs page is that?

Reply:
This is highly confusing, because https://docs.giantswarm.io/guides/tenant-cluster-labelling/ does not mention Azure at all

Reply:
yes, exactly. 

I got it from Adrian's PR: https://github.com/giantswarm/docs/pull/667/files it was merged, I guess it wasn't released yet?

Reply:
I removed the labelling item

Reply:
Anyway, can we just remove it from the announcement to block it?